charts/templates/secret.yaml

```diff
@@ -1,3 +1,9 @@
+{{- $adminPassword := (randAlphaNum 8) | b64enc | quote }}
```

Comment:
Is there a reason the logic is structured like this and not just an if/then/else?

Reply:
yes, variable adminPassword needs to be defined first. Hence defining and setting value here and not in the else block.